src/mochaVErrorReporter.ts

```diff
@@ -0,0 +1,46 @@
+import { Runner, utils as MochaUtils } from "mocha";
+import { VError } from "verror";
+
+var stackFilter = MochaUtils.stackTraceFilter();
```

Comment:
can this be made into a static member as well?

Reply:
Fixed